### DISCUSSION TITLE
quoted AS clause returning wrong results in 3.4.0

### DISCUSSION THREAD

**Discussion by bweinman** [pawelsalawa/sqlitestudio#4571] [General]
Queries that use a quoted alias (AS clause) return the alas as data 

```
drop table if exists foo;
create table foo (a text, b text);
insert into foo values ('this', 'that');
insert into foo values ('this', 'that');
select a as "foo bar", b from foo;
```

First time through, it sometimes works. Subsequent times it returns "foo bar" as the data for the a column. 

I've confirmed this works as expected in command line sqlite3. 

I use SQLiteStudio to teach online at Linkedin Learning. My students are having a lot of trouble with this bug. 

<img width="1019" alt="image" src="https://user-images.githubusercontent.com/480448/203157697-ffb73461-5abe-4e78-bdcb-70839427fb90.png">

**Comment by pawelsalawa** [comment]
It was already reported in #4549. It's already fixed and pending release in 3.4.1 (soon).
